src/shared.ts

```diff
@@ -5,6 +5,8 @@ export * from './lib/utilities/index';
 export * from './lib/permissions/clients/index';
 export * from './lib/permissions/interfaces';
 
+export * from './lib/proposals/interfaces';
```

Comment:
Why do interfaces need to be exported from shared? they should be importable on the frontend using `type` just fine